src/utils/GitHub.test.js

```diff
@@ -69,4 +72,18 @@ describe('GitHub', () => {
       expect(res.status).toEqual(httpOK)
     })
   })
+  describe('get models from github', () => {
```

Comment:
Once the mocks are updated to contain real shape and responses, these test assertions should also be updated.